packages/storybook/src/generators/migrate-9/calling-storybook-cli.ts

```diff
@@ -38,7 +43,9 @@ export function callUpgrade(schema: Schema): 1 | Buffer {
       bodyLines: [
         `🚨 The Storybook CLI failed to upgrade your @storybook/* packages to the latest version.`,
         `Please try running the sb upgrade command manually:`,
-        `${pm.exec} storybook@latest upgrade`,
+        `${pm.exec} ${
```

Comment:
I wonder if there could be an option to upgrade to the @next version based on a flag maybe? We normally go through full prerelease cycles (alpha, beta, RC) and during that phase we normally ask people to already upgrade their projects